agenta-web/src/components/pages/evaluations/evaluators/EvaluatorCard.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 import {EvaluatorConfig, JSSTheme} from "@/lib/Types"
 import {DeleteOutlined, EditOutlined} from "@ant-design/icons"
-import {Card, Tag, Typography} from "antd"
+import {Card, Tag, Typography, Modal} from "antd"
```

Comment:
Can we please remove this unused import now?

Reply:
Hi, okay . Only Modal was unused right?.

Reply:
Updated through web. Don't mind, currently my laptop is not with me.